### DISCUSSION TITLE
Which game are playable?

### DISCUSSION THREAD

**Discussion by Windowstore** [skyline-emu/skyline#242] [General]
Please someone tell me which games are playable on skyline emulator especially Pokemon sword and shield

**Comment by sauradipdebnath** [comment]
Metroid

**Reply by Windowstore** [reply]
Can you tell me which version

**Reply by sauradipdebnath** [reply]
Metroid dread

**Reply by Windowstore** [reply]
but can't playable

**Reply by mlgprorektm8** [reply]
try teh demo then

**Comment by JerryM33** [comment]
![Screenshot_20220109_224926](https://user-images.githubusercontent.com/97395782/148688105-b88b2e92-4f40-4cbd-bdd9-e97a5c05803e.jpg)

**Comment by HalogenGam3r** [comment]
Currently **only** Sonic Mania is supported. More games are coming soon.

**Comment by jmtuch** [comment]
Playable:
Another World 
Bastion 
Blossom Tales 
Carrion 
Celeste 
Croc's World
Crypt of the NecroDancer
Hammerwatch
Miles & Kilo
Miner Warfare 
Mercenary Kings 
NeuroVoider
Owlboy 
Scourge Bringer 
Shovel Knights King of Cards 
Sonic Mania 
Steam World Dig 2 
Super Crush K.O 
The King of Fighters R-2
Tiny Barbarian DX
VVVVVV

**Reply by Bobombn64** [reply]
vvvvvv is playable too

**Reply by DerroRK** [reply]
Mercenary Kings playable too

**Comment by Someg1993** [comment]
Steam World Dig 2 is loading very good but it needs more fps in order to be playable.

**Comment by Gsxrpitts61** [comment]
> Please someone tell me which games are playable on skyline emulator especially Pokemon sword and shield

 They both play now on v48

**Comment by Gsxrpitts61** [comment]
Yes they both are playable on my sd888
